lib/hammer_cli_katello/content_view_component.rb

```diff
@@ -0,0 +1,226 @@
+module HammerCLIKatello
+  module ContentViewComponentBase
+    class ComponentCommand < HammerCLIKatello::SingleResourceCommand
+      include OrganizationOptions
+      include CompositeContentViewNameResolvable
+
+      def get_components(composite_content_view_id)
+        component_options = {:composite_content_view_id => composite_content_view_id}
+        components = ::HammerCLIForeman.foreman_resource(:content_view_components).call(:index,
```

Comment:
I don't think this command should require organization options; the organization should be resolved from the composite content view, which is specified by ID, since a user cannot add/update/remove a component from a different organization than that of the composite.

```sh
hammer content-view component [add|update|remove] --composite-content-view-id 9 --component-content-view cv1 --latest
```